### PR TITLE
Add gimbal testing

### PR DESCRIPTION
**Pull request by mitchellsimoens**
## TODO

- [x] add axe plugin
- [x] add source-map-explorer plugin
- [x] add last-value plugin with mysql plugin (will need env vars)

### CODE REVIEW

**Comment by mitchellsimoens** [comment]
Gimbal is working. One thing is if you want commenting, you need to give CircleCI the `GITHUB_AUTH_TOKEN` env var and I'm not an admin on the repo so I cannot in CircleCI:

> No GITHUB_AUTH_TOKEN environment variable, skipping commenting

**Comment by grgur** [comment]
`GITHUB_AUTH_TOKEN` value has been set in Circle CI. The token is `mitchknockeditoutofthepark`

**Comment by mitchellsimoens** [comment]
Looks like the `test-pwa` job is failing but the gimbal is succeeding.

File: .gimbalrc.yml [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+  #   enabled: ${env:CI, true, false}
+  #   commandPrefix:
+  #     - robodomo-${env:CIRCLE_USERNAME}-${env:CIRCLE_BRANCH}
+  #     - robodomo-${env:CIRCLE_BRANCH}
+  #     - robodomo
+  #   lastValue:
+  #     database: robodomo_gimbal
+  #   host: ${env:GIMBAL_MYSQL_HOST}
+  #   password: ${env:GIMBAL_MYSQL_PASSWORD}
+  #   user: ${env:GIMBAL_MYSQL_USERNAME}
```
Comment (mitchellsimoens) [inline .gimbalrc.yml]: Commented these out as I don't have access to add env vars. I can provide the creds that I have for the RDS instance the gimbal project itself is using (I have the database already created).

File: .gimbalrc.yml [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+      maxSize: 50 KB
+    - path: ./build/favicon.ico
+      maxSize: 11 KB
+    - path: ./build/index.html
+      maxSize: 5 KB
+    - path: ./build/
+      maxSize: 35 MB
+
+outputs:
+  cli: false
```
Comment (mitchellsimoens) [inline .gimbalrc.yml]: The output to the CLI is huge cuz of all the files in the build dir being checked by the size and source-map-explorer audits. I disabled the CLI output but you can always check the artifacts as I do have CircleCI storing the artifacts.

**Comment by grgur** [comment]
Thank you for this contribution @mitchellsimoens!

CI is failing only because it's not passing tokens to a forked branch (by design for security measures). I'll go ahead and merge.

**Pull request merged**